.lintstagedrc

```diff
@@ -1,8 +1,7 @@
 {
   "*.+(js|ts|tsx)": [
     "eslint",
-    "jest --findRelatedTests",
-    "jest --passWithNoTests"
```

Comment:
are we going to drop this?

Reply:
unintentional - restored in [64a25](https://github.com/marshmallow-insurance/smores-react/pull/2045/commits/64a25440dceacadbe36831f6b74e921de6626fa0)